### PR TITLE
[PHP 8.4] intl changes part 1.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by devnexen** [comment]
for #3872

File: reference/intl/dateformatter/create.xml [hunk 1]
```diff
@@ -156,6 +156,15 @@
       </row>
      </thead>
      <tbody>
+      <row>
+       <entry>8.4.0</entry>
```
Comment (nielsdos) [inline reference/intl/dateformatter/create.xml:160]: Should also be in the error section of the page

File: reference/intl/dateformatter/create.xml [hunk 2]
```diff
@@ -156,6 +156,15 @@
...
+       <entry>
+        <para>
+         A <classname>Value</classname> is thrown if
```
Comment (nielsdos) [inline reference/intl/dateformatter/create.xml:163]:
```suggestion
         A <exceptionname>ValueError</exceptionname> is thrown if
```

File: reference/intl/numberformatter/create.xml [hunk 3]
```diff
@@ -103,6 +103,15 @@
      </row>
     </thead>
     <tbody>
+     <row>
```
Comment (nielsdos) [inline reference/intl/numberformatter/create.xml:106]: Same remarks as for dateformatter

File: reference/intl/dateformatter/create.xml [hunk 4]
```diff
@@ -156,6 +156,15 @@
       </row>
      </thead>
      <tbody>
+      <row>
+       <entry>8.4.0</entry>
+       <entry>
+        <para>
```
Comment (nielsdos) [inline reference/intl/dateformatter/create.xml:162]: Please use simpara

File: reference/intl/resourcebundle/get.xml [hunk 5]
```diff
@@ -73,6 +72,38 @@
...
+      <row>
+       <entry>&Version;</entry>
+       <entry>&Description;</entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>8.4.0</entry>
+       <entry>
+        <para>
```
Comment (nielsdos) [inline reference/intl/resourcebundle/get.xml:90]: simpara please
Also add this to the error section of the page

File: reference/intl/resourcebundle/get.xml [hunk 6]
```diff
@@ -73,6 +72,38 @@
...
+         A <classname>TypeError</classname> is thrown if
```
Comment (nielsdos) [inline reference/intl/resourcebundle/get.xml:91]: Use exceptionname instead of classname in these descriptions

**Review by nielsdos — CHANGES_REQUESTED** [review]

File: reference/intl/dateformatter/create.xml (hunk 2 above)
Comment (nielsdos) [reply reference/intl/dateformatter/create.xml]: Not addressed yet (notice Value**Error**) and mistake copied in error section (check all places for this mistake)

File: reference/intl/dateformatter/create.xml [hunk 7]
```diff
@@ -144,6 +144,14 @@
   </para>
  </refsect1>
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <para>
+   A <exceptionname>ValueError</exceptionname> is thrown if
+   <parameter>locale</parameter> is invalid.
+  </para>
```
Comment (Girgias) [inline reference/intl/dateformatter/create.xml:152]:
```suggestion
  <simpara>
   A <exceptionname>ValueError</exceptionname> is thrown if
   <parameter>locale</parameter> is invalid.
  </simpara>
```

File: reference/intl/numberformatter/create.xml [hunk 8]
```diff
@@ -92,6 +92,14 @@
   </para>
  </refsect1>
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <para>
+   A <exceptionname>ValueError</exceptionname> is thrown if
+   <parameter>locale</parameter> is invalid.
+  </para>
```
Comment (Girgias) [inline reference/intl/numberformatter/create.xml:100]:
```suggestion
  <simpara>
   A <exceptionname>ValueError</exceptionname> is thrown if
   <parameter>locale</parameter> is invalid.
  </simpara>
```

File: reference/intl/resourcebundle/get.xml [hunk 9]
```diff
@@ -73,6 +72,52 @@
   </para>
  </refsect1>
 
+ <refsect1 role="errors">
+  &reftitle.errors;
+  <para>
```
[the remark this answers is not in the document]
Comment (Girgias) [reply reference/intl/resourcebundle/get.xml:77]: I don't think we have a consistent style tbh, I have used multiple `<simpara>` tags in general. But I don't mind either style.

File: reference/intl/resourcebundle/get.xml [hunk 10]
```diff
@@ -73,6 +72,52 @@
...
+   if <parameter>index</parameter> is a <type>string</type>
+   and is empty or is a <type>int</type> and does not
+   fit into a 32 bit integer type.
+  </para>
+ </refsect1>
+
+ <refsect1 role="changelog">
+  &reftitle.changelog;
+  <para>
+   <informaltable>
```
Comment (Girgias) [inline reference/intl/resourcebundle/get.xml:92]: The wrapping `<para>` tag here is useless.

File: reference/intl/resourcebundle/get.xml [hunk 11]
```diff
@@ -73,6 +72,52 @@
...
+       </entry>
+      </row>
+     </tbody>
+    </tgroup>
+   </informaltable>
+  </para>
+ </refsect1>
+
+
  <refsect1 role="examples">
```
Comment (Girgias) [inline reference/intl/resourcebundle/get.xml:121]: Nit:
```suggestion
 </refsect1>

 <refsect1 role="examples">
```

**Review by Girgias — CHANGES_REQUESTED** [review]
Thank you, XML markup nits

**Review by nielsdos — APPROVED** [review]
I don't see problems anymore

**Pull request merged**